src/views/Table.vue

```diff
@@ -0,0 +1,187 @@
+<!--
+Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+  <div>
+    <CylcObjectMenu />
+    <div class="c-table">
+      <table-component
+        :tasks="tasks"
+        :hoverable="false"
+        :activable="false"
+        :multiple-active="false"
+        :min-depth="1"
```

Comment:
We can remove `min-depth`, `multiple-active`, `activable`, `hoverable`. Leaving just the `tasks` prop.

Reply:
My file looks quite different:
```
      <table-component
        :tasks="tasks"
       props :min-depth="1"
        ref="table0"
        key="table0"
      ></table-component>
      ```
      There's no `hoverable` or similar props there

Reply:
In which case I think your local branch is ahead or behind this remote branch? 😕

Reply:
(Or you had a different branch checked out?  What this PR shows *is* your tableview branch (in your GitHub fork of cylc-ui) so your file can't look different, so long as you have the right branch checked out and haven't made any local changes since pushing it to GitHub).

Reply:
Hmm quite puzzled by this one to be honest. I haven't really changed anything locally and this issue was already happeni g on Friday...

Reply:
It'll be quite hard to diagnose this one without looking at your working copy. We can have a video chat tomorrow to see if we can figure out what's going on.

Otherwise, I think `git fetch --all` and then `git status` should display if your branch is up to date or not to the remote branch (you can also set the remote branch as `git branch --set-upstream-to=upstream/master` or something like that).

Reply:
Table Component no longer has the props mentioned above, but only has `tasks` prop so I believe this is also solved.